PluginGenerator/PluginBuilder.cs

```diff
@@ -275,6 +276,18 @@ protected string FindPluginKey()
             return pluginKey;
         }
 
+        protected string FindPackageName()
```

Comment:
Previously subclasses didn't have a way to determine the package name (should probably be called "PackageNamespace"). The rules plugin now needs this info as it uses the package namespace to make the names of the rules and sqale files unique.